app/pages/Auction/RepairBid.js

```diff
@@ -50,17 +51,19 @@ export default class RepairBid extends Component {
           }
         }
         disabled={this.state.isCorrect}
-      />      
+      />
     );
   }
 
   processValue = async (val) => {
     const value = val.match(/[0-9]*\.?[0-9]{0,6}/g)[0];
-    if (value * consensus.COIN > consensus.MAX_MONEY)
+    const dollarydoos = parseFloat(value) * consensus.COIN;
+    if (dollarydoos > consensus.MAX_MONEY)
       return;
     this.setState({value: value});
-    if (parseFloat(value))
-      return this.verifyBid(value);
+    if (val === "")
+      return;
```

Comment:
Mine as well move this to the top of the function I think.

Reply:
This needs to be after the `setState`. Otherwise, user can't clear the input.

Reply:
great, thanks